synapse/storage/databases/main/metrics.py

```diff
@@ -171,7 +172,7 @@ def _count_messages(txn):
 
         return await self.db_pool.runInteraction("count_messages", _count_messages)
 
-    async def count_daily_sent_messages(self):
+    async def count_daily_sent_messages(self) -> int:
         def _count_messages(txn):
```

Comment:
Some of the inner methods here don't have type hints still, that's fine we can always add them later, but wanted to point it out!

Reply:
We can leave it to the future then. 👍